### PR TITLE
Fix tiny typo, usuage -> usage

### PR DESCRIPTION
**Pull request by forslund**
Fixes a tiny type noticed when running the example code.

Relates-To: OLPEDGE-342
Signed-off-by: Åke Forslund <ake.forslund@scania.com>

### CODE REVIEW

**Comment by andescu** [comment]
@forslund Thanks for noticing the typo and taking the time to propose the fix.
To be able to merge you will need to sign-off the commit via git and use a ticket reference.
You can use OLPEDGE-342 for this. So the commit message should be something like:

```
Fix tiny typo, usuage -> usage.

Fixes a tiny type noticed when running the example code.

Relates-to: OLPEDGE-342
Signed-off-by: ...
```

Then we should be able to merge.

**Comment by forslund** [comment]
There updated according to request :)

**Comment by andescu** [comment]
@forslund  Sorry, I made a mistake.
The ticket reference needs to be `Relates-To`, not `Relates-to`. Please amend the commit message once more and we should be able to merge.

**Comment by forslund** [comment]
Ok, let's see if this is better.

**Review by andescu — APPROVED** [review]

**Review by mykhailo-kuchma — APPROVED** [review]

**Pull request merged**
